crates/router/src/connector/tsys.rs

```diff
@@ -461,6 +474,12 @@ impl ConnectorIntegration<api::Execute, types::RefundsData, types::RefundsRespon
         &self,
         req: &types::RefundsRouterData<api::Execute>,
     ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
+        let connector_router_data = tsys::TsysRouterData::try_from((
+            &self.get_currency_unit(),
+            req.request.currency,
+            req.request.refund_amount,
+            req,
+        ))?;
         let req_obj = tsys::TsysRefundRequest::try_from(req)?;
```

Comment:
Hey @zakhaev26, 
To address 
```
error[E0412]: cannot find type `T` in this scope
--> crates/router/src/connector/tsys/transformers.rs:494:33
 |
494 | impl<F> TryFrom<&TsysRouterData<T>::RefundsRouterData<F>> for TsysRefundRequest {
 |      -                          ^
 |      |
 |      similarly named type parameter `F` defined here
 
```
 change
```suggestion
        let req_obj = tsys::TsysRefundRequest::try_from(connector_router_data)?;
```
and do adequate changes to the `TryFrom` of `RefundsRouterData` in transformer.rs file